vips/vip-410/bscmainnet.ts

```diff
@@ -0,0 +1,270 @@
+    vToken: "0xAeB0FEd69354f34831fe1D16475D9A83ddaCaDA6",
+    rewardsDistributor: ARBITRUM_XVS_REWARDS_DISTRIBUTOR_CORE,
+    blocksPerMonth: ARBITRUM_BLOCKS_PER_MONTH,
+  },
+  {
+    chainId: LzChainId.arbitrumone,
+    newAllocation: ethers.utils.parseEther("239"),
+    isSupplierAllocation: true,
+    isBorrowerAllocation: false,
+    vToken: "0x68a34332983f4Bf866768DD6D6E638b02eF5e1f0",
+    rewardsDistributor: ARBITRUM_XVS_REWARDS_DISTRIBUTOR_CORE,
+    blocksPerMonth: ARBITRUM_BLOCKS_PER_MONTH,
+  },
+  {
+    chainId: LzChainId.arbitrumone,
+    newAllocation: ethers.utils.parseEther("319"),
+    isSupplierAllocation: true,
+    isBorrowerAllocation: false,
+    vToken: "0xaDa57840B372D4c28623E87FC175dE8490792811",
+    rewardsDistributor: ARBITRUM_XVS_REWARDS_DISTRIBUTOR_CORE,
+    blocksPerMonth: ARBITRUM_BLOCKS_PER_MONTH,
+  },
+  {
+    chainId: LzChainId.arbitrumone,
+    newAllocation: ethers.utils.parseEther("479"),
+    isSupplierAllocation: false,
+    isBorrowerAllocation: true,
+    vToken: "0xB9F9117d4200dC296F9AcD1e8bE1937df834a2fD",
+    rewardsDistributor: ARBITRUM_XVS_REWARDS_DISTRIBUTOR_CORE,
+    blocksPerMonth: ARBITRUM_BLOCKS_PER_MONTH,
+  },
+  {
+    chainId: LzChainId.arbitrumone,
+    newAllocation: ethers.utils.parseEther("479"),
+    isSupplierAllocation: false,
+    isBorrowerAllocation: true,
+    vToken: "0x7D8609f8da70fF9027E9bc5229Af4F6727662707",
+    rewardsDistributor: ARBITRUM_XVS_REWARDS_DISTRIBUTOR_CORE,
+    blocksPerMonth: ARBITRUM_BLOCKS_PER_MONTH,
+  },
+  {
+    chainId: LzChainId.arbitrumone,
+    newAllocation: ethers.utils.parseEther("1020"),
+    isSupplierAllocation: true,
+    isBorrowerAllocation: false,
+    vToken: "0x39D6d13Ea59548637104E40e729E4aABE27FE106",
+    rewardsDistributor: ARBITRUM_XVS_REWARDS_DISTRIBUTOR_LST,
+    blocksPerMonth: ARBITRUM_BLOCKS_PER_MONTH,
+  },
+  {
+    chainId: LzChainId.arbitrumone,
+    newAllocation: ethers.utils.parseEther("0"),
+    isSupplierAllocation: false,
+    isBorrowerAllocation: false,
+    vToken: "0x9df6B5132135f14719696bBAe3C54BAb272fDb16",
+    rewardsDistributor: ARBITRUM_XVS_REWARDS_DISTRIBUTOR_LST,
+    blocksPerMonth: ARBITRUM_BLOCKS_PER_MONTH,
+  },
+  {
+    chainId: LzChainId.arbitrumone,
+    newAllocation: ethers.utils.parseEther("0"),
+    isSupplierAllocation: false,
+    isBorrowerAllocation: false,
+    vToken: "0x246a35E79a3a0618535A469aDaF5091cAA9f7E88",
+    rewardsDistributor: ARBITRUM_XVS_REWARDS_DISTRIBUTOR_LST,
+    blocksPerMonth: ARBITRUM_BLOCKS_PER_MONTH,
+  },
+];
+
+const commands = emissions.map(emission => {
+  const speed = emission.newAllocation.div(emission.blocksPerMonth);
+  return {
+    target: emission.rewardsDistributor,
+    signature: "setRewardTokenSpeeds(address[],uint256[],uint256[])",
+    params: [
+      [emission.vToken],
+      [emission.isSupplierAllocation ? speed : 0],
+      [emission.isBorrowerAllocation ? speed : 0],
+    ],
+    dstChainId: emission.chainId,
+  };
+});
```

Comment:
We could reduce the commands, grouping them by destination chain and RewardsDistributor contract:
```suggestion
}).reduce((accumulator, currentValue) => {
    const command = accumulator.find(
      it => it.target == currentValue.target && it.dstChainId == currentValue.dstChainId,
    );
    if (!command) {
      accumulator.push(currentValue);
    } else {
      command.params[0].push(currentValue.params[0][0]);
      command.params[1].push(currentValue.params[1][0]);
      command.params[2].push(currentValue.params[2][0]);
    }
    return accumulator;
  }, []);
```